modules/Behavior.js

```diff
@@ -0,0 +1,146 @@
+    'before-dispose': '_dispose',
+    'render:before-attach-tracked-views': 'attachTrackedViews',
+    'render:begin': 'prerender',
+    'render:complete': 'postrender',
+    'initialize:begin':  '_preinitialize',
+    'initialize:complete': '_postinitialize'
+  };
+
+  /**
+   * Allows abstraction of common view logic into separate object
+   *
+   * @module Torso
+   * @class  Behavior
+   * @method constructor
+   * @author  deena.wang@vecna.com
+   */
+  var Behavior = Cell.extend({
+    /**
+     * @property cidPrefix of Behaviors
+     * @type {String}
+     */
+    cidPrefix: 'b',
+
+    /**
+     * @method constructor
+     * @param options {Object}
+     * @param options.view {Backbone.View} that Behavior is attached to
+     */
+    constructor: function(options) {
+      options = options || {};
+      if (!options.view) {
+        throw new Error('Torso Behavior constructed without options.view');
+      }
+      this.view = options.view;
+      this.cid = _.uniqueId(this.cidPrefix);
+      this.__bindLifecycleMethods();
+      this.__bindEventCallbacks();
+      Cell.apply(this, arguments);
+    },
+
+    /**
+     * Registers defined lifecycle methods to be called at appropriate time in view's lifecycle
+     *
+     * @method __bindLifecycleMethods
+     * @private
+     */
+    __bindLifecycleMethods: function() {
+      _.each(eventMap, function(callback, event) {
+        this.listenTo(this.view, event, this[callback]);
+      }, this);
+    },
+
+    /**
+     * Adds behavior's event handlers to view
+     * Behavior's event handlers fire on view events but are run in the context of the behavior
+     *
+     * @method __bindEventCallbacks
+     * @private
+     */
+    __bindEventCallbacks: function() {
+      var behaviorEvents = _.result(this, 'events');
+      var viewEvents = this.view.events;
+
+      if (!viewEvents) {
+        if (!behaviorEvents) {
+          return;
+        } else {
+          viewEvents = {};
+        }
+      }
+
+      var namespacedEvents = this.__namespaceEvents(behaviorEvents);
+      var boundBehaviorEvents = this.__bindEventCallbacksToBehavior(namespacedEvents);
+
+      if (_.isFunction(viewEvents)) {
+        this.view.events = _.wrap(_.bind(viewEvents, this.view), function(viewEventFunction) {
+          return _.extend(boundBehaviorEvents, viewEventFunction());
+        });
+      } else if (_.isObject(viewEvents)) {
+        this.view.events = _.extend(boundBehaviorEvents, viewEvents);
+      }
+    },
+
+    /**
+     * Namespaces events in event hash
+     *
+     * @method __namespaceEvents
+     * @param eventHash {Object} to namespace
+     * @return {Object} with event namespaced with '.behavior' and the cid of the behavior
+     * @private
+     */
+    __namespaceEvents: function(eventHash) {
+      // coped from Backbone
+      var delegateEventSplitter = /^(\S+)\s*(.*)$/;
+      var namespacedEvents = {};
+      var behaviorId = this.cid;
```

Comment:
I went with a cid because I don't think the behavior should know or care what the view decides to name the behavior. I could add the view's name for the behavior as an additional namespace, but I'm not sure what value that provides. Example?